okapi/core.php

```diff
@@ -1902,6 +1904,26 @@ public static function fix_oc_html($html)
 
         return $html;
     }
+
+    function php_ini_get_bytes($setting)
+    {
+        $setting = ini_get($setting);
+        $setting = str_replace('K', '*1024', $setting);
+        $setting = str_replace('M', '*1024*1024', $setting);
+        $setting = eval('return '.$setting.';');
```

Comment:
(btw, I still haven't read all of your emails, sorry - I decided I will first take a look at the diff)